src/modeler/AutoRest.Swagger/Validation/OperationParametersValidation.cs

```diff
@@ -11,7 +11,6 @@ namespace AutoRest.Swagger.Validation
     public class OperationParametersValidation : TypedRule<SwaggerParameter>
     {
         private const string SubscriptionId = "subscriptionid";
-        private const string ApiVersion = "api-version";
```

Comment:
Just add a to-do saying we need to add api-version in the check some time in the future when we enable the single-swagger spec mode for validation.